src/utils/consts/consts_vars.go

```diff
@@ -0,0 +1,45 @@
+package consts
```

Comment:
do we really need one package called utils?

https://go.dev/blog/package-names

and do we need one module called consts? where will theses consts used? i think that theses const must stay proximity where are used...

https://www.approxion.com/the-principle-of-proximity/

@esequielvirtuoso @julianogalgaro @ebbauer

Reply:
We use it several times in the project.
That's why I've separated it into a specific package.

Reply:
Regarding the package utils, they can be removed.
I've placed all the utility codes in it to keep the project tree organized.